TuneUp/TuneUpWindowViewModel.cs

```diff
@@ -224,6 +219,15 @@ private void CurrentWorkspaceModel_EvaluationStarted(object sender, EventArgs e)
         private void CurrentWorkspaceModel_EvaluationCompleted(object sender, Dynamo.Models.EvaluationCompletedEventArgs e)
         {
             IsRecomputeEnabled = true;
+            RaisePropertyChanged(nameof(ProfiledNodesCollection));
+            RaisePropertyChanged(nameof(ProfiledNodes));
+            ProfiledNodesCollection.Dispatcher.Invoke(() =>
```

Comment:
it's a little funny to get the dispatcher from here, but should be fine in most cases.